ludwig/models/modules/h3_encoders.py

```diff
@@ -495,7 +477,7 @@ def __init__(
             dropout=dropout,
             initializer=initializer,
             regularize=regularize,
-            reduce_output=None
+            reduce_output='None'
         )
 
         self.recurrent_stack = RecurrentStack(
```

Comment:
here if the recurrent stack has additional parameteres that are not surfaced we should probably add them to the init of h3rnn. If those are all the parameters, then we are good.

Reply:
While working to surface the recurrent stack parameters, I noticed this
https://github.com/jimthompson5802/ludwig/blob/8a069c1ce8e8994825999c5de5ece478d70300d2/ludwig/models/modules/h3_encoders.py#L392-L406.  Line 402 has `droput=False`.  From what I can tell, this should be a float between 0.0 and 1.0.  In theory, `False` numerically evaluates to 0.0. I did a quick skim how `dropout` is used.  From what I can tell, it is used only as a numerical attribute.  

Any issue with changing from `dropout=False` to `dropout=0.0`?

Reply:
Yes it should be changed to `dropout_rate` like everywhere else, and it should be 0.0 by default. Also, the surfacing thing should be done for the FClayers too, and also in the date feature is needed. Didn't realize it when you sbmitted the date PR, but in general the principle is: surface as many parameters from the layers used inside the encoder as possible (or as makes sense). For the sequence feature encoders for instance I spent quite some time to make sure all the relevant parameters are surfaced, we should make sure the same happens in all other features.

Reply:
added remaining recurrent stack parameters, changed `dropout` to `dropout_rate`.